code/src/main/java/frc/robot/RobotConfig.java

```diff
@@ -39,6 +40,11 @@
 
         public final int visionTrackButtonPort = XboxController.Button.kB.value;
 
+        public final int leftHeading = 270;
+        public final int rightHeading = 90;
+        public final int forwardHeading = 0;
+        public final int backHeading = 270;
+
```

Comment:
rename these to `fieldWestButtonAngle`, `fieldEastButtonAngle`, `fieldNorthButtonAngle`, and `fieldSouthButtonAngle`